_includes/integrations/angular-quick-start.md

```diff
@@ -122,14 +114,40 @@ This procedure requires:
               { "glob": "**/*", "input": "node_modules/tinymce/plugins", "output": "/tinymce/plugins/" }
             ]
             ```
-        4. Update the editor configuration to include the `base_url` and `suffix` options.
+            
+            If you are using tinymce-angular 4.0.0 or later, add the {{site.productname}} core script as well
+            ```json
+            { "input": "node_modules/tinymce/tinymce.min.js", "output": "/tinymce/tinymce.min.js" }
+            ```
+        3a. If you are using tinymce-angular 4.0.0 or later, the recommended approach for providing the {{site.productname}} core script is to have tinymce-angular lazy load it. This is achieved through the usage of dependency injection. Add a dependency provider to your module that imports the EditorModule using the `TINYMCE_SCRIPT_SRC` token. No additional configuration is required as {{site.productname}} will load assets such as plugins relative to this location.
+
+            ```js
+            import { EditorModule, TINYMCE_SCRIPT_SRC } from '@tinymce/tinymce-angular';
+            /* ... */
+            @NgModule({
+              /* ... */
+              imports: [
+                EditorModule
+              ],
+              providers: [
+                { provide: TINYMCE_SCRIPT_SRC, useValue: 'tinymce/tinymce.min.js' }
```

Comment:
This should probably be `/path/to/tinymce.min.js`, as that's what's used in other examples for `tinymceScriptSrc`. In saying that though, this likely be the correct value given the above step copies it across.